rxgroups/src/main/java/com/airbnb/rxgroups/ObservableGroup.java

```diff
@@ -186,14 +208,14 @@ public void call(ManagedObservable<?> managedObservable) {
     }
 
     /**
-     * Returns an existing {@link Observable} for the provided {@code tag}. Does not change the
-     * locked
-     * status of this {@link ObservableGroup}. If it is unlocked, and the Observable has already
-     * emitted events, they will be immediately delivered. If it is locked then no events will be
+     * Returns an existing {@link Observable} for the provided {@code tag}.
+     * <p> Does not change the locked status of this {@link ObservableGroup}.
+     * If it is unlocked, and the Observable has already emitted events,
+     * they will be immediately delivered. If it is locked then no events will be
      * delivered until it is unlocked.
      */
     public <T> Observable<T> observable(AutoResubscribingObserver<? super T> observer) {
-        return observable(observer, null);
+        return observable(observer, observer.getTag());
```

Comment:
I see one problem with using the observer tag as the second tag instead of null - the observer tag is hidden from the user so they don't know what it is, and technically they could subscribe another observable with that tag to this observer.

This seems really unlikely, but null seems safer.